bots/tests-scan

```diff
@@ -366,21 +351,16 @@ def update_status(api, revision, context, last, changes):
         return False
     return True
 
-def cockpit_tasks(api, update, contexts, repo, pull_data, pull_number, sha, amqp):
+def cockpit_tasks(api, update, branch_contexts, repo, pull_data, pull_number, sha, amqp, is_external):
     results = []
-    branch_contexts = { }
-    for (context, branches) in contexts.items():
-        for branch in branches:
-            if branch not in branch_contexts:
-                branch_contexts[branch] = [ ]
-            branch_contexts[branch].append(context)
-
     pulls = []
+    contexts = set(itertools.chain(*branch_contexts.values()))
+
     if pull_number or pull_data:
         # Avoid processing pull requests with the same number on external
         # repositories: if repo is None, the api is querying the repo the pull
         # request belongs to
-        if repo is None:
+        if not is_external:
```

Comment:
Ha, I guess it is enough to check for `cockpit-project/cockpit`

Reply:
Or not. This was called with `None` directly from main and then with repo set for external projects. To keep it working when cloned into weldr for example it cannot check for `c-p/c`.